src/Spryker/Zed/RabbitMq/RabbitMqConfig.php

```diff
@@ -52,4 +62,41 @@ public function getExchangeNameBlacklist()
     {
         return ['/^amq./', ExchangeInfo::AMQP_DEFAULT_EXCHANGE_NAME];
     }
+
+    /**
+     * @return string
+     */
+    public function getVirtualHost()
+    {
+        return $this->get(RabbitMqConstants::RABBITMQ_VIRTUAL_HOST);
+    }
+
+    /**
+     * @return string
+     */
+    protected function getApiHost()
+    {
+        return $this->get(RabbitMqConstants::RABBITMQ_API_HOST);
+    }
+
+    /**
+     * @return int
+     */
+    protected function getApiPort()
+    {
+        return $this->get(RabbitMqConstants::RABBITMQ_API_PORT);
+    }
+
+    /**
+     * @return string
+     */
+    public function getApiPermissionsAdminUrl()
+    {
+        return sprintf(
+            'http://%s:%s/api/permissions/%s/admin',
```

Comment:
we dont support/need https here btw?

Reply:
Not at the moment

Reply:
OK, just saying that would be easy to configure with a either a flag or just making the URL aware of the schema by using `'//:...'` urls :) easy

Reply:
This happens in the console, where there's no `//:` as far as I know.